docs/basics.md

```diff
@@ -0,0 +1,55 @@
+# The Basics
+
+## Result
+
+All input passed through Membrane will return as part of the data object [Result](result.md).
+
+All Results can have these three properties:
+
+### Value
+
+This is what your input will have become after passing through Membrane.
+
+### Result
+
+This is to inform you whether Membrane has considered the input valid or invalid.
```

Comment:
I extended upon it a bit more so it now says this

> No result occurs if only filters have been applied to the data. It has not been flagged as invalid, 
> however no validator has been applied to validate it.
> 
> The isValid() method considers anything valid as long as it is not invalid. It is recommended to use this
> method instead of relying on the Result property alone.